### ISSUE TITLE
:before and :after psuedos do not merge with ::before and ::after

### ISSUE THREAD

**Issue by clshortfuse** [clean-css/clean-css#1084]
### Environment

* clean-css version - 4.2.1:
* node.js version - 10.16.0:
* operating system: Windows 10 / Ubuntu 18

### Configuration options

```
var CleanCSS = require('clean-css');
new CleanCSS({
   level: 2
})
```

### Input CSS

```
.a:before { color: red }
.a::before { background-color: orange }
```

### Actual output CSS

```
.a:before{color:red}.a::before{background-color:orange}
```

### Expected output CSS

```
.a:before{color:red;background-color:orange}
```

The single colon `:before` from CSS2 and double colon `::before` from CSS3 are effectively the same and can be merged. You also save a character by using `:before` instead of `::before`. Same applies to `:after`. 

https://developer.mozilla.org/en-US/docs/Web/CSS/::before#Syntax

**Comment by jakubpawlowicz** [comment]
Won't fix as clean-css is going into [maintenance mode](https://github.com/clean-css/clean-css/discussions/1209).
